main.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"cloud.google.com/go/storage"
+	"context"
+	"flag"
+	"fmt"
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+	"gopkg.in/yaml.v2"
+	"log"
+	"os"
+	"time"
+
+)
+// Config struct for Directory, Bucket and GCS credential path details
+type Config struct {
+	Dir          string `yaml:"dir"`
+	DaysLimit    int `yaml:"daysLimit"`
+}
+
+func NewConfig(configFile string) ([]Config, error) {
```

Comment:
This function no need to be Public
```suggestion
func newConfig(configFile string) ([]Config, error) {
```